docs/getting-started/quick-start.md

```diff
@@ -0,0 +1,74 @@
+# Quick Start
```

Comment:
Looks like duplicated
https://github.com/aquasecurity/trivy/blob/a67955a0040a47eb363d8f5c9a8c44096215dd0c/docs/index.md#get-trivy

Reply:
I am not sure why Dimitry's work is getting merged in...